source/Engine.cpp

```diff
@@ -747,7 +747,17 @@ void Engine::Step(bool isActive)
 			info.SetBar("fuel", flagship->Fuel(), fuelCap * .01);
 		else
 			info.SetBar("fuel", flagship->Fuel());
+		const double scale = .2 + 1.8 / (.001 * flagship->Position().Length() + 1);
```

Comment:
What are these numbers for? Are they just arbitrary?

Reply:
This is the same calculation used in `Ship::DoGeneration` as part of calculating ramscoop and solar energy collection.
I suppose it might be better to move it to its own method somewhere.